policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/ApplicationConfiguration.java

```diff
@@ -50,29 +50,25 @@
     @Bean
     @ConditionalOnProperty(prefix = "population", name = "policy", havingValue = "std")
     public StdPolicyConfiguration policyConfiguration() {
-        LOGGER.info("Standard Policy Configuration Instantiated");
         return new StdPolicyConfiguration();
     }
 
     @Bean
     @ConditionalOnProperty(prefix = "population", name = "policy", havingValue = "std")
-    public StdPolicyCacheWarmerFactory policyCacheWarmerFactory() {
-        LOGGER.info("Standard Policy Cache Warmer Instantiated");
-        return new StdPolicyCacheWarmerFactory();
+    public StdPolicyPrepopulationFactory policyCacheWarmerFactory() {
```

Comment:
```suggestion
    public StdPolicyPrepopulationFactory policyPrepopulationFactory() {
```
Is it worth renaming these beans so that they mirror the class name?